src/reference/00-Getting-Started/07A-Task-Graph.md

```diff
@@ -0,0 +1,324 @@
+---
+out: Task-Graph.html
+---
+
+  [Basic-Def]: Basic-Def.html
+  [Scopes]: Scopes.html
+  [Make]: https://en.wikipedia.org/wiki/Make_(software)
+  [Ant]: http://ant.apache.org/
+  [Rake]: https://ruby.github.io/rake/
+
+Task graph
+----------
+
+This page explains `build.sbt` definition in more detail.
+It assumes you've read [.sbt build definition][Basic-Def] and
+[scopes][Scopes].
+
+### What's the point of the build.sbt DSL?
+
+As we saw before, a [build definition][Basic-Def] consists of subprojects
+with a map called `settings` describing the subproject.
+There's more to the story.
+
+What the `Setting` sequence encodes is tasks and the dependencies among them,
+similar to [Make][Make] (1976), [Ant][Ant] (2000), and [Rake][Rake] (2003).
+
+#### Intro to Make
```

Comment:
I agree that explaining how SBT works should be the first priority. sbt may be the first build tool the reader encounters, and the current layout might give the impression that you need experience with other build tools to understand sbt. That's not true at all! Comparison with other build tools could belong to a separate "Coming from Make/Rake/..." section.